ci-operator/config/cloud-bulldozer/e2e-benchmarking/cloud-bulldozer-e2e-benchmarking-master.yaml

```diff
@@ -0,0 +1,34 @@
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: e2e
+    tag: latest
+images:
+- dockerfile_path: prow/Dockerfile
+  to: e2e-benchmarking
+promotion:
+  to:
+  - namespace: e2e
+    tag: latest
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.15"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: unit
+  commands: echo "test"
```

Comment:
This a WIP PR?

Reply:
No its a actual PR. I want e2e onboarded to PROW first, verify its working in e2e and then take tests in a different PR.